deployments/properties/google-cloudrun-docker.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Build and Deploy to Cloud Run",
+  "description": "Build a Docker container, publish it to Google Artifact Registry, and deploy to Google Cloud Run.",
+  "creator": "Google Cloud",
+  "iconName": "google-cloud",
+  "categories": ["Deployment", "Containers", "Dockerfile", "Cloud Run", "Serverless"]
```

Comment:
@verbanicm do the categories of "Cloud run" and "Serverless" mean that there will be specific files for these in user's repo. If yes, then we can help enhance a repo analysis logic which we have today that detects certain files and prioritizes such workflows when showing to user.
Let me know if that is the case and I can help with more details on how to make such enhancement.